frontend/app/configuration/Config.scala

```diff
@@ -152,6 +152,9 @@ object Config {
   val googleAnalyticsTrackingId = config.getString("google.analytics.tracking.id")
 
   val youtubeMembershipVerificationId = config.getString("youtube.membership.verification.id")
+  val youtubeUrl = config.getString("youtube.url")
+
+  val googleplusUrl = config.getString("googleplus.url")
```

Comment:
I would put it in it's own file (`Social.scala`), which could live under `frontend/app/configuration` if you like! They're not really configuration, just constants.